examples/models/object_detection/maskrcnn_rafiki.py

```diff
@@ -0,0 +1,5122 @@
+import json
```

Comment:
Thanks for you suggestions, I would modify this file recently. Since I am really busy now, after graduation, I need to renew my pass, also search for a job, It may take some time. 
It's my first time to contribute to open source project,  it's fun and thanks for your review.